classes/services/queryBuilders/PKPStatsGeoQueryBuilder.php

```diff
@@ -111,6 +111,11 @@ public function getSum(array $groupBy = []): Builder
         return $q;
     }
 
+    /**
+     * Consider/add application specific queries
+     */
+    abstract protected function _getAppSpecificQuery(Builder &$q): void;
```

Comment:
Could also implement this in the base class so you don't need an empty implementation in the app class.